core/include/cubos/core/al/audio_device.hpp

```diff
@@ -109,9 +102,12 @@ namespace cubos::core::al
         public:
             virtual ~Source() = default;
 
+            /// @brief  Creates a source
+            static std::shared_ptr<Source> create();
```

Comment:
Same thing as the buffer - this function shouldn't exist - the user should call `AudioDevice::create`